README.md

```
@@ -1,85 +1,63 @@
-This project is open source and contributions are welcome. Please refer to `CONTRIBUTING.md` (to be created) for guidelines.
-
-## License
-
-This project will be licensed under the [AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html#license-text) license.
\ No newline at end of file
+# 📻 Open Ham Awards
+**The lightweight, open-source contest and award management platform for Amateur Radio clubs.**
+
+*Project Status: Early Development 🚧 | [Join the Waitlist for Early Access](tbd)*
+
+### **The Problem**
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)